src/include/sof/math/decibels.h

```diff
@@ -0,0 +1,17 @@
+
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
+ */
+
+#ifndef DECIBELS_H
+#define DECIBELS_H
+
+#include <stdint.h>
+
+int32_t exp_fixed(int32_t x); /* Input is Q5.27, output is Q16.16 */
+int32_t db2lin_fixed(int32_t x); /* Input is Q8.24, output is Q12.20 */
```

Comment:
The decibels.h should define the fractional Qxy notations of input and output parameters. It helps make the code more portable for other possible implementations for the functions.

Reply:
done